README.md

```diff
@@ -1,7 +1,488 @@
-## Sagemaker Containers
+# SageMaker Containers
 
-This support code is used for making machine learning frameworks (e.g. MXNet, TensorFlow) run on Amazon SageMaker. 
+SageMaker Containers contains common functionality necessary to create a container compatible with SageMaker. It can be simply used by any container by just installing the module:
```

Comment:
Can you include a table of contents? I enjoyed using this to generate this from the headers: https://github.com/jonschlinkert/markdown-toc

You might need to change some headers to make the table of contents turn out well.